src/metalperformanceshaders.cs

```diff
@@ -8,7 +9,7 @@ namespace XamCore.MetalPerformanceShaders {
 
 	// MPSImageConvolution.h
 
-	[iOS (9,0)]
+	[iOS (9,0)][Mac (10,13)]
```

Comment:
do it work on 32bits ? iirc metal itself was 64 bits only
this means the attribute is incomplete, i.e. -> `[Mac (10,13, onlyOn64 : true)]`